test/org/sagebionetworks/bridge/play/controllers/UserManagementControllerTest.java

```diff
@@ -87,6 +116,38 @@ public void before() throws Exception {
         when(request.headers()).thenReturn(map);
         doReturn(null).when(controller).getMetrics();
     }
+
+    @Test
+    public void signInForAdmin() throws Exception {
```

Comment:
For branch coverage, we probably also want a test for the case where a non-admin calls this API.

Reply:
Added